packages/design-system/src/components/OcResourceIcon/OcResourceIcon.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { shallowMount } from '@vue/test-utils'
+import { AVAILABLE_SIZES } from '../../helpers/constants'
+import { OcResourceIcon } from '..'
+
+const resources = ['file', 'folder', 'space']
+
+function getWrapper(props) {
+  return shallowMount(OcResourceIcon, {
```

Comment:
Please return an object with key `wrapper` and your `shallowMount` call as value, i.e.
```js
return {
  wrapper: shallowMount(...
```
for consistency with other tests in `web`.